Kitodo/src/main/java/org/kitodo/production/forms/LdapServerForm.java

```diff
@@ -69,9 +73,15 @@ public String newLdapServer() {
      */
     public String save() {
         try {
-            ServiceManager.getLdapServerService().saveToDatabase(this.ldapServer);
+            String securitySecret = ConfigCore.getParameterOrDefaultValue(ParameterCore.SECURITY_SECRET);
+
+            if (StringUtils.isNotBlank(securitySecret)) {
+                ldapServer.setManagerPassword( AESUtil.encrypt(ldapServer.getManagerPassword(), securitySecret) );
+            }
+
+            ServiceManager.getLdapServerService().saveToDatabase(ldapServer);
             return usersPage;
-        } catch (DAOException e) {
+        } catch ( Exception e ) {
```

Comment:
Catching every exception here is not a good idea. Fetch only this exception which you would handle here and let all other go on.